plugin/src/main/resources/io/jenkins/plugins/remotingkafka/GlobalKafkaConfiguration/config.jelly

```diff
@@ -1,12 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
     <f:section title="${%Remoting Kafka Configuration}">
-        <f:entry field="connectionURL" title="${%Connection URL}">
+        <f:entry field="zookeeperURL" title="${%Zookeeper URL}">
             <f:textbox/>
         </f:entry>
-        <f:entry field="zookeeperURL" title="${%Zookeeper URL}">
+        <f:validateButton
+                title="${%Test Zookeeper Connection}" progress="${%Testing...}"
+                method="testZookeeperConnection" with="zookeeperURL" />
+        <f:entry field="brokerURL" title="${%Kafka Broker URL}">
+            <f:textbox/>
+        </f:entry>
+        <f:validateButton
+                title="${%Test Broker Connection}" progress="${%Testing...}"
+                method="testBrokerConnection" with="brokerURL" />
+        <!--<f:entry field="username" title="${%Username}">-->
+            <!--<f:textbox/>-->
+        <!--</f:entry>-->
+        <f:entry title="${%Credentials}" field="credentialsId">
```

Comment:
It would be nice to support simple non-authenticated mode for users. There could be two options: 

* Hide all security configuration under `optionalProperty` (will require moving data to a separate class)
* Do the same via `optional` (I would recommend `optionalProperty` tho)
* Allow keeping fields empty